gflownet/policy/multihead_tree.py

```diff
@@ -0,0 +1,502 @@
+from typing import Optional
```

Comment:
At risk of being very annoying: docstring. Seriously, just a one- or two-liner would do.